src/components/services/user.ts

```diff
@@ -0,0 +1,5 @@
+import todos from '../../api/todos';
+
+export function getTodoById(id: number) {
+  return todos.filter(todo => todo.userId === id) || null;
```

Comment:
The function name `getTodoById` suggests that it should return a single todo by its `id`, but it currently filters todos by `userId`. Consider renaming the function to `getTodosByUserId` for clarity, or adjust the logic to filter by `todo.id` if that was the original intention.